### PR TITLE
Adding a no. of releases function

### PR DESCRIPTION
**Pull request by Tarun-Acharya**
For every release, we can find the no. of releases between the current and latest version. This can be summed under a loop and printed at the end.

### CODE REVIEW

**Comment by nasirhjafri** [comment]
@Tarun-Acharya Thanks for the feature. Would you like to add a new command line argument and hook this function with under main function [here](https://github.com/nasirhjafri/libyear/blob/master/libyear/libyear)?

**Comment by Tarun-Acharya** [comment]
I am not sure how to add and parse an argument, but I can write the looping code under the main function to print the total releases along with the lib years, you can later edit that part under new argument.

**Comment by Tarun-Acharya** [comment]
I have just made a new commit adding hooking the function onto the main.

**Comment by Tarun-Acharya** [comment]
Can u merge my request?

**Comment by nasirhjafri** [comment]
> Can u merge my request?

Would you mind removing the code from the main function? I'll add it later with command line arguments.

File: libyear/libyear [hunk 1]
```diff
@@ -18,14 +18,15 @@ def main():
...
+    total_releases = 0
     for req in requirements:
         name, version, version_lt = get_requirement_name_and_version(req)
         if not name:
             continue
 
         if not version and not version_lt:
             continue
-
+        total_releases+=get_no_of_releases(name, version)
```
Comment (nasirhjafri) [inline libyear/libyear:29]: Either add a command-line argument or remove the changes from this file for now.

Comment (Tarun-Acharya) [reply libyear/libyear]: Yes, I have removed the changes.

**Comment by nasirhjafri** [comment]
Thanks @Tarun-Acharya :)

**Pull request merged**